src/commands/importDocuments.ts

```diff
@@ -0,0 +1,134 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { NewDocument } from 'documentdb';
+import * as fse from 'fs-extra';
+import * as vscode from 'vscode';
+import { AzureTreeDataProvider, IAzureParentNode, parseError } from 'vscode-azureextensionui';
+import { DocDBCollectionTreeItem } from '../docdb/tree/DocDBCollectionTreeItem';
+import { DocDBDocumentsTreeItem } from '../docdb/tree/DocDBDocumentsTreeItem';
+import { ext } from '../extensionVariables';
+import { MongoCollectionTreeItem } from '../mongo/tree/MongoCollectionTreeItem';
+
+export async function importDocuments(tree: AzureTreeDataProvider, uris: vscode.Uri[] | undefined, collectionNode: IAzureParentNode<MongoCollectionTreeItem | DocDBCollectionTreeItem> | undefined): Promise<void> {
+    if (!uris) {
+        uris = await askForDocuments();
+    }
+    let ignoredUris: vscode.Uri[] = []; //account for https://github.com/Microsoft/vscode/issues/59782
+    uris = uris.filter((uri) => {
+        if (uri.fsPath.endsWith('.json')) {
```

Comment:
Should this feature support jsonc files?

Reply:
Good question. I expect using fse.readJson might curtail this to only support json. Let me test this.

Reply:
fse.readJSOn does not  support jsonc.